features/admin.users.v1/components/user-profile.tsx

```diff
@@ -2230,10 +2232,21 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
         );
     };
 
+    const resolveRequiredValue = (schema: ProfileSchemaInterface,
```

Comment:
Let's add a doc comment for this method.

https://github.com/wso2/identity-apps/blob/master/docs/write-code/CODE_QUALITY.md#typescript-doc-comments